### ISSUE TITLE
Target 404 doesn't report broken?

### ISSUE THREAD

**Issue by kingthorin** [technote-space/broken-link-checker-action#136]
## Describe the bug: バグの概要
Setup the action to run against a list of URLs, if one of the targets is broken it doesn't fail or report the link broken.

## To Reproduce: 再現手順
Setup the action as follows:
```
on:
  schedule:
    - cron: 0 0 1 * * # run monthly
  workflow_dispatch:

name: Broken Link Check
jobs:
  check:
    name: Broken Link Check
    runs-on: ubuntu-latest
    steps:
      - name: Broken Link Check
        uses: technote-space/broken-link-checker-action@v2
        with:
          TARGET: |
            - https://example.org
            - https://owasp.org/www-community/fail
          RECURSIVE: false
```

## Expected behavior: 期待する動作
The owasp.org link 404s so the action should fail and an issue should be opened.

## Screenshots: スクリーンショット
![image](https://user-images.githubusercontent.com/7570458/135881563-cb588341-8428-4909-8a9d-2d8538da339f.png)
![image](https://user-images.githubusercontent.com/7570458/135881743-79896b6c-c642-4a6e-be8a-47c67ecc819e.png)

## Operating environment: バグが発生した環境
 - Version of software: technote-space/broken-link-checker-action@v2
 - OS: [e.g. Windows10]: ubuntu-latest
 - Browser: [e.g. Chrome, Safari] N/A
 - etc.

**Comment by kingthorin** [comment]
Any plans to address this?

**Comment by technote-space** [comment]
使用方法が違うため
TARGET は array は対応していない
そもそも TARGET に指定したページ内のリンクのチェックを行うアクションなので別のアクションを使用すべき

**Comment by kingthorin** [comment]
What other information is needed?
